app/Http/Controllers/Api/ManagerApiController.php

```diff
@@ -68,14 +68,14 @@ public function showAuthenticated()
     /**
      * Update the specified resource in storage.
      *
-     * @param  \App\Http\Requests\UpdateManager $request Incoming Form Request.
-     * @param  \App\Models\Manager              $manager Incoming Manager.
+     * @param  \App\Http\Requests\UpdateManagerApi $request Incoming Form Request.
```

Comment:
Curious why this was renamed?

Reply:
Asked by Tristan, I believe to prevent confusion between the new UpdateManagerProfileRequest.php file.